pages/forms/homepage/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" >
+    <meta property="og:title" content="We Life Post" />
+    <meta property="og:type" content="page.png" />
+    <meta property="og:url" content="" />
+    <meta property="og:description" content="WeLife: Website sharing and discovery platform. Join our community and show your digital creations to the world. Explore creative projects and find inspiration. Join today!">
+    <link rel="shorcut icon" href="icon.ico">
+    <title>Forms practice</title>
+  </head>
+  <body>
+    <header>
+      <title>We Life</title>
+      <div class="header">
+        <a href="../login/index.html">Login</a>
+        <a href="../registration/index.html">Register</a>
+      </div>
+    </header>
+    <main class="home-page" >
+      <h1 id="web-title">Lorem Ipsum</h1>
+      <p id="web-descripcion">"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt."</p>
+      <a href="../gon/index.html">
+        <div for="2" class="post">
+          <h4>Gon</h4>
+          <p>Hunter x Hunter sassortment of characters is undoubtedly one of the series strongest points. Most fans appreciate the characters and their wild dynamics more than any other aspect concerning the series. The characters are, after all, the common denominator between all the essential attributes that build up the Hunter x Hunter that fans love and care for.</p>
+          <p>2021-05-10</p>
+        </div>
+        <div>
+          <h5>Comments</h5>
+          <p>7</p>
+        </div>
+      </a>
+      <a href="../hxh/index.html">
+        <div for="3" class="post">
+          <h4>HXH</h4>
+          <p>The story of Hunter x Hunter centers on a young man named Gon, who discovers that his father, who was told he was dead, is actually alive. Gon's father is a legendary hunter named Ging, a hunter whose recognition and worth is known to everyone.</p>
+          <p>2023-05-04</p>
+        </div>
+        <div>
+          <h5>Comments</h5>
+          <p>8</p>
+        </div>
+      </a>
+      <a href="../gon-images/index.html">
+        <div for="4" class="post">
+          <h4>Gon Images.</h4>
+          <p>Images and small descriptions about one of the most beloved characters, <b>Gon</b></p>
+          <p>2024-05-02</p>
+        </div>
+        <div class="comments">
+          <h5>Comments</h5>
+          <p>15</p>
+        </div>
+      </a>
+      <form action="../registrer/index.html">
```

Comment:
This could be better, but I understand the intention here. We can leave it as it is for now and change it later.

Reply:
Perfect.